### PR TITLE
update(engine) : Update Compatibility with hopeit.engine 0.25.0

### PR DESCRIPTION
**Pull request by pcanto-hopeit**
This pull request updates the hopeit.aws.s3 plugin to be compatible with hopeit.engine version 0.25.0. It includes changes to ensure that the plugin works seamlessly with the latest version of the engine.

Changes:

    Updated compatibility with hopeit.engine version 0.25.0.

Breaking Changes:

    This update is not backward compatible with previous versions of hopeit.engine.

### CODE REVIEW

File: release-notes.rst [hunk 1]
```diff
@@ -1,14 +1,25 @@
 Release Notes
 =============
 
+Version 0.2.0b1
```
Comment (leosmerling-hopeit) [inline release-notes.rst]: Usually release notes of beta release are set under the version 0.2.0, no need to add b1 in the release notes. Then all changes are cummulative until the final 0.2

File: release-notes.rst [hunk 2]
```diff
@@ -1,14 +1,25 @@
...
+_____________
+- hopeit.aws.s3 
+
+   - Updated to be compatible with `hopeit.engine` version 0.25.0.
+   
+   BREAKING CHANGES
+   ================
+
+   - This version is not backward compatible with previous version of the `hopeit.engine`.
```
Comment (leosmerling-hopeit) [inline release-notes.rst]: "This version is not backwards compatible with previous versions of `hopeit.engine`"

**Review by leosmerling-hopeit — APPROVED** [review]

File: plugins/aws/s3/requirements.txt [hunk 3]
```diff
@@ -1,2 +1,2 @@
-hopeit.engine>=0.24.2,<1
-aioboto3>=13.0.0
\ No newline at end of file
+hopeit.engine==0.25.0b1
```
Comment (leosmerling-hopeit) [inline plugins/aws/s3/requirements.txt]: Dont pin the version use >=

Comment (pcanto-hopeit) [reply plugins/aws/s3/requirements.txt]: fixed in 3749053

File: release-notes.rst [hunk 4]
```diff
@@ -10,7 +10,7 @@ _____________
    BREAKING CHANGES
    ================
 
-   - This version is not backward compatible with previous version of the `hopeit.engine`.
+   - This version is not backward compatible with previous version of `hopeit.engine`.
```
Comment (leosmerling-hopeit) [inline release-notes.rst:13]: "backwards compatible" (with "s")

**Pull request merged**